xpdschema/schemas/run_start.json

```diff
@@ -0,0 +1,178 @@
+      "type": "object",
+      "description": "dictionary of sample phase information"
+    },
+    "sample_maker":{
+      "type": "array",
+      "description": "list of sample maker names "
+    },
+    "sa_uid":{
+      "type": "string",
+      "description": "auto-generated uid for Sample object"
+    },
+    "cif_name":{
+      "type": "array",
+      "description": "list of possible structure filenames"
+    },
+    "collaborators":{
+      "type": "array",
+      "description": "list of collaborator names"
+    },
+    "database_id":{
+      "type": "object",
+      "description": "dictionary of structure uid to different database"
+    },
+    "lead_experimenter":{
+      "type": "array",
+      "description": "list of lead experimenter names"
+    },
+    "notes":{
+      "type": "string",
+      "description": "memorable note for sample"
+    },
+    "tags":{
+      "type": "array",
+      "description": "list of short tags user sets up for future search"
+    },
+    "sp_plan_name":{
+      "type": "string",
+      "description": "name of ScanPlan"
+    },
+    "sp_args":{
+      "type": "array",
+      "description": "positional arguments passed to ScanPlan"
+    },
+    "sp_kwargs":{
+      "type": "object",
+      "description": "keyword arguments passed to ScanPlan"
+    },
+    "sp_uid":{
+      "type": "string",
+      "description": "auto-generated uid for ScanPlan object"
+    },
+    "sp_time_per_frame":{
+      "type": "number",
+      "description": "acquisition time per frame in second"
+    },
+    "sp_num_frames":{
+      "type": "number",
+      "description": "number of frames being summed per trigger"
+    },
+    "sp_requested_exposure":{
+      "type": "number",
+      "description": "user requested exposure time in second"
+    },
+    "sp_computed_exposure":{
+      "type": "number",
+      "description": "program computed exposure time"
+    },
+    "sp_type":{
+      "type": "string",
+      "description": "ScanPlan type of xpdacq"
+    },
+    "sp_startingT":{
+      "type": "number",
+      "description": "starting temperature of xpdAcq Tramp ScanPlan"
+    },
+    "sp_endingT":{
+      "type": "number",
+      "description": "ending temperature of xpdAcq Tramp ScanPlan"
+    },
+    "sp_requested_Tstep":{
+      "type": "number",
+      "description": "user requested temperature step of xpdAcq Tramp ScanPlan"
+    },
+    "sp_computed_Tstep":{
+      "type": "number",
+      "description": "program computed temperature step of xpdAcq Tramp ScanPlan"
+    },
+    "sp_Nsteps":{
+      "type": "number",
+      "description": "program computed number of steps of xpdAcq Tramp ScanPlan"
+    },
+    "calibration_md":{
+      "type": "object",
+      "description": "dictionary of calibration parameters linked by xpdAcq"
+    },
+    "mask":{
+      "type": "array",
+      "description": "sparse representation of the mask generated by auto-mask function"
+    },
+    "energy_calibration_client_uid":{
```

Comment:
Previously the plan was to have two keys `energy_calibraion_uid` and `is_energy_calibration`. I realized that we coud put them into one key if we denoted clients and servers. The data with the `*_server_uid` key contains the data referred to by the key, in this case the header with  `energy_calibration_server_uid` would contain the data needed to perform energy calibration analysis. The `energy_calibration_client_uid` containing header would need the data from the server referenced by their shared uid. Please also see the section in the [readme](https://github.com/xpdAcq/xpdSchema/pull/1/files#diff-04c6e90faac2675aa89e2176d2eec7d8R5)